012-quarkus-kafka-streams/src/test/java/io/quarkus/qe/resources/KafkaContainerHelper.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.qe.resources;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.GenericContainer;
+
+public class KafkaContainerHelper<SELF extends GenericContainer<SELF>> extends GenericContainer<SELF> {
```

Comment:
I don't think this is a helper, but an extension to the Strimzi Kafka container. A proper name could be `BaseStrimziKafkaContainer`.
Being said that, do we need this class? From my point of view, we don't need to check the compatible kafka versions and the doStart method could be done in the security implementations (it's just about to duplicate a single line).

Reply:
We have removed this class, because currently, we support up to Kafka 2.7.0